css/index.css

```diff
@@ -3,13 +3,223 @@ This imports the `reset.css` file */
 @import url('./reset.css');
 
 /* For testing purposes, delete this code */
+
 body {
-  background-color: #000fff;
+  background-color: hsl(222, 26%, 31%);
```

Comment:
Please let try making use of .root for all the color it will make the code more clearer and understandable